Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -763,6 +763,43 @@ - (ORKOrderedTask *)makeSelectionSurveyTask {
         [steps addObject:step];
     }
     
+    
+    {
+        /*
+         A text question with single-line text entry and a number pad for keyboard.
+         */
+        ORKTextAnswerFormat *format = [ORKAnswerFormat textAnswerFormat];
+        format.multipleLines = NO;
+        format.keyboardType = UIKeyboardTypeNumberPad;
```

Comment:
I understand we need to test `keyboardType`, but this example could be misleading, developer may think this is the recommended way to accept numeric input instead of `ORKNumbericAnswerFormat`. Can we change this keyboardType something else, e.g. `UIKeyboardTypeURL`, and update the question text accordingly?